src/utils/asset.js

```diff
@@ -90,3 +101,11 @@ export const getAssetIcon = (asset) => {
     }
   }
 }
+
+export const getExplorerTransactionHash = (asset, hash) => {
+  switch (asset) {
+    case 'NEAR':
+      return hash.split('_')[0]
```

Comment:
Yeah It is explorer specific. The problem is that in the Near package the transaction hash is stored in the form TXHASH_ address as this is the right format.   Not sure if it could be included in CAL